vlib/v/checker/tests/orm_using_non_struct_field_in_order_by_error.out

```diff
@@ -1,8 +1,14 @@
 vlib/v/checker/tests/orm_using_non_struct_field_in_order_by_error.vv:14:29: error: V ORM: `User` structure has no field with name `database`.
 2 possibilities: `id`, `username`.
-   12 |
+   12 | 
```

Comment:
Extra space?

Reply:
This output is fully generated by V.

Reply:
Does the .vv file have an extra space?  That would cause this...

Reply:
No. And the file is auto-formatted.

Reply:
Ok, time for a bug report against V.  It shouldn't output excess spaces.  :-\